packages/react-select/src/Select.tsx

```diff
@@ -1078,11 +1093,16 @@ export default class Select<
     key: Key,
     props: StylesProps<Option, IsMulti, Group>[Key]
   ) => {
-    const base = defaultStyles[key](props as any);
+    const { unstyled } = this.props;
```

Comment:
I've made `unstyled` only available on the parent component (for now) because:

- It minimises the scope of this PR
- What happens if `unstyled` on the parent component is set to `true`, but on the inner component it's set to `false`?
- We can easily add it to the inner components later. If we added it now, and decided to change it later, it would be a breaking change.